### ISSUE TITLE
Troubles with using Length(Qubit[]) function in Jupyter Notebook

### ISSUE THREAD

**Issue by yung-chiller** [MicrosoftDocs/quantum-docs-pr#1022]
I'm trying to execute my Q# code from Python code via Jupyter Notebook. 
Python code is following:
```
import qsharp
qsharp.packages.add("Microsoft.Quantum.MachineLearning::0.11.2006.403")
qsharp.reload()
```
The error is:
```
error CS0234: Тип или имя пространства имен "_25aba78bf5b747e0ab8eef471c954521_Length" не существует в пространстве имен "Microsoft.Quantum.Core" (возможно, отсутствует ссылка на сборку).
```
(Approximate translation: type or namespace ```_25aba78bf5b747e0ab8eef471c954521_Length``` doesn't exist in namespace ```Microsoft.Quantum.Core```)
In my code there're several lines like ```let x = Length(qs);``` where ```qs``` is a ```Qubit[]``` array. Erasing them or using expressions ```Length(qs)``` in ```for``` cycles only (like ```for (i in 0..Length(qs)-1)```) doesn't give this error.

**Comment by rmshaffer** [comment]
@yung-chiller, thanks for reporting! Can you try updating your qsharp and iqsharp to the latest version (should be 0.12.20072031), and also update your Python code to load the matching version of the `Microsoft.Quantum.MachineLearning` package? I believe this is a bug that was fixed in 0.12.2007* versions, so it would be great if you could confirm that!

**Comment by geduardo** [comment]
Closing due to lack of response. @yung-chiller you can reopen if the problem persists.
